crates/host/Cargo.toml

```diff
@@ -7,11 +7,8 @@ authors = ["thedavidmeister", "thedavidmeister@gmail.com"]
 edition = "2021"
 
 [dependencies]
-wasmer = { version = "=4.3.6", optional = true, default-features = false }
-wasmer-middlewares = { version = "=4.3.6", optional = true, default-features = false }
-
-# Temporarily include a fork of wasmer from the git branch 'wamr', until it is officially released in wasmer v5
-hc-wasmer = { version="=4.3.6-hc.1", optional = true, default-features = false }
+wasmer = { version = "5.0.2", default-features = false }
```

Comment:
Should this stay pinned? Really depends how good we think wasmer are at managing breaking changes

Reply:
It looks like we've been pinning the wasmer version since the first release of this crate (when wasmer versioning was at 0.x). I'm not sure what the rationale was, but it doesn't look like it was in response to a specific case where wasmer broke semver.

Idk maybe we should remove the pin and re-add it if there is an issue